python3/bin/perfmon

```diff
@@ -525,7 +525,7 @@ class VariableConfig:
         try:
             name = xmldoc.getElementsByTagName("name")[0].getAttribute("value")
         except IndexError:
-            raise XmlConfigException("variable missing 'name' tag")
+            raise XmlConfigException("variable missing 'name' tag") from None
```

Comment:
The whole point of the error is to chain the exceptions:
```
        except IndexError as e:
            raise XmlConfigException("variable missing 'name' tag") from e
```

Reply:
Code updated.

Reply:
I understand we need to chain the exceptions.
But the `from None` breaks the chain actually.